frontend/src/constants/API.js

```diff
@@ -21,4 +21,5 @@ export const MINE_TSF_REQUIRED_DOCUMENTS = "/documents/required?category=MINE_TA
 export const EXPECTED_DOCUMENT = "/documents/expected";
 export const MINE_TENURE_TYPES = "/mines/mine_tenure_type_codes";
 export const MINE_TYPES = "/mines/mine_types";
+export const MINE_TYPES_DETAILS = "/mines/mine_types/details";
```

Comment:
Please note, there is an open PR right now that converts all of the snake_case URLs to kebab-case.

Reply:
ah, okay. good to know.